packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -10,6 +10,7 @@ import {
     semanticColor,
     border,
 } from "@khanacademy/wonder-blocks-tokens";
+import {focusStyles} from "@khanacademy/wonder-blocks-styles";
```

Comment:
We'll need to include the styles package as a dependency in the link package, and update the `tsconfig-build.json` file!